packages/vscode-extension/README.md

```diff
@@ -1,64 +1,85 @@
-![NetSuite Logo](resources/oracle_netsuite_logo.png)
+<p align="center"><a href="#"><img width="250" src="resources/oracle_netsuite_logo.png"></a></p>
 
-# SuiteCloud Extension for Visual Studio Code
-The SuiteCloud extension allows you to customize your SuiteCloud Development Framework (SDF) projects for NetSuite.
+# <font size="7"> <span style="color:gray"> **SuiteCloud Extension for Visual Studio Code** </font> </span>
+SuiteCloud Extension for Visual Studio Code is part of the SuiteCloud Software Development Kit (SuiteCloud SDK), a set of tools to customize your NetSuite accounts. The SuiteCloud Extension for Visual Studio Code allows you to customize your SuiteCloud Development Framework (SDF) projects for NetSuite. The available SDF projects include **SuiteApp projects** — self-contained, standalone projects that you can publish in the SuiteApp Marketplace for other users to download and install, and **Account Customization Projects** — intended for customizations on accounts you own, such as your production, development, and sandbox accounts.
 
-> 💡 To read more about SDF, visit [NetSuite's Help Center.](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_4702656040.html)
+Read more about SuiteCloud SDK and SDF in <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=chapter_156026236161.html" rel="noopener noreferrer" target=blank>Getting Started with SuiteCloud SDK</a> and <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_4702656040.html" rel="noopener noreferrer" target=blank>SuiteCloud Development Framework Architecture</a>.
 
-The following types of SDF projects are available:
-- SuiteApp projects — self-contained, standalone projects that you can publish in the SuiteApp Marketplace for other users to download and install.
-  > 💡 To read more about SuiteApp projects, visit [NetSuite's Help Center.](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_1509931104.html)
-- Account customization projects — intended for customizations on accounts you own, such as your production, development, and sandbox accounts.
-  > 💡 To read more about account customization projects, visit [NetSuite's Help Center.](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_1510680449.html)
+## <font size="6"> <span style="color:gray"> Installation prerequisites </font> </span>
+The following software is required to work with SuiteCloud Extension for Visual Studio Code:
+- Node.js version 14.16.0 LTS
+- Oracle JDK version 11
```

Comment:
Dont forget to add the bullet point vscode version:   vscode 1.57 or higher.

Reply:
Added.